definitions/procedures/content/prepare.rb

```diff
@@ -3,12 +3,14 @@ class Prepare < ForemanMaintain::Procedure
     metadata do
       description 'Prepare content for Pulp 3'
       for_feature :pulpcore
+      param :append_output, 'Preserve all output lines', :flag => true, :default => false
```

Comment:
As per the redmine details and considering overall user exp I feel we should invert this flag? By default it will show the progress details and if required one can silent it? If you agree then I also think the param can be something like `quite`?